packages/@lwc/compiler/src/transformers/transformer.ts

```diff
@@ -23,6 +23,7 @@ export interface TransformResult {
     code: string;
     map: unknown;
     warnings?: CompilerDiagnostic[];
+    cssScopeTokens?: string[];
```

Comment:
Why is this optional?

Reply:
Only the template compiler returns it, not the style compiler or babel compiler.